src/main/java/com/hierynomus/ntlm/messages/NtlmAuthenticate.java

```diff
@@ -15,32 +15,34 @@
  */
 package com.hierynomus.ntlm.messages;
 
-import java.nio.charset.StandardCharsets;
 import com.hierynomus.protocol.commons.EnumWithValue;
 import com.hierynomus.protocol.commons.buffer.Buffer;
 import com.hierynomus.protocol.commons.buffer.Endian;
 
+import java.nio.charset.StandardCharsets;
+
 import static com.hierynomus.ntlm.functions.NtlmFunctions.unicode;
 
 /**
  * [MS-NLMP].pdf 2.2.1.3 AUTHENTICATE_MESSAGE
  */
 public class NtlmAuthenticate extends NtlmPacket {
     private static byte[] EMPTY = new byte[0];
-
     private byte[] lmResponse;
     private byte[] ntResponse;
     private byte[] userName;
     private byte[] domainName;
     private byte[] workstation;
     private byte[] encryptedRandomSessionKey;
     private long negotiateFlags;
+    private boolean useMIC;
```

Comment:
Though MIC is an acronym, please rename to `useMic`, and also the field below to `mic` (as `MIC` is a constant name)